docs/artefacts/codecs.md

```diff
@@ -97,3 +97,37 @@ For **gauge** there is also the field `action` which might be `add` if the value
 ### yaml
 
 En- and decodes [YAML](https://yaml.org).
+
+### syslog
+
+En- and decodes syslog messages (both, the standard IETF format and the old BSD format).
+A typical syslog message as follows:
+
+```text
+<165>1 2021-03-18T20:30:00.123Z mymachine.example.com evntslog - ID47 [exampleSDID@32473 iut=\"3\" eventSource=\"Application\" eventID=\"1011\"] BOMAn application event log entry..."
+```
+
+get translates to:
+
+```json
+{
+  "severity": "notice",
+  "facility": "local4",
+  "hostname": "mymachine.example.com",
+  "appname": "evntsog",
+  "msg": "BOMAn application event log entry...",
+  "procid": null,
+  "msgid": "ID47",
+  "protocol": "RFC5424",
+  "protocol_version": 1,
+  "structured_data": {
+              "exampleSDID@32473" :
+              [
+                {"iut": "3"},
+                {"eventSource": "Application"},
+                {"eventID": "1011"}
+              ]
+            }
+  "timestamp": 1_616_099_400_123_000_000_u64
```

Comment:
while this is really nice in rust sadly json doesn't support it :( the `_` and `_u64` won't work here